StoreMessageContentActivity/__tests__/handler.test.ts

```diff
@@ -253,11 +296,1075 @@ describe("getStoreMessageContentActivityHandler", () => {
+      aRetrievedProfile.fiscalCode
+    ]);
+
+    const documentId = makeServicesPreferencesDocumentId(
+      aRetrievedProfile.fiscalCode,
+      "01234567890" as NonEmptyString,
+      manualProfileServicePreferencesSettings.version as NonNegativeInteger
+    );
+    expect(findServicePreferenceMock).toHaveBeenCalled();
+    expect(findServicePreferenceMock).toHaveBeenCalledTimes(1);
+    expect(findServicePreferenceMock).toHaveBeenCalledWith([
+      documentId,
+      aRetrievedProfile.fiscalCode
+    ]);
+  });
+
+  it("should return a failure if message sender service exists and has INBOX disabled in user service preference (MANUAL SETTINGS)", async () => {
+    findLastProfileVersionByModelIdMock.mockImplementationOnce(() =>
+      taskEither.of(
+        some({
+          ...aRetrievedProfile,
+          servicePreferencesSettings: manualProfileServicePreferencesSettings,
+          blockedInboxOrChannels: {
+            "01234567890": [BlockedInboxOrChannelEnum.INBOX]
+          }
+        })
+      )
+    );
+
+    findServicePreferenceMock.mockImplementationOnce(
+      ([modelId, partitionKey]) =>
+        taskEither.of<CosmosErrors, Option<ServicePreference>>(
+          some({
+            ...anEnabledServicePreference,
+            isInboxEnabled: false,
+            version: manualProfileServicePreferencesSettings.version
+          })
+        )
+    );
+
+    const storeMessageContentActivityHandler = getStoreMessageContentActivityHandler(
+      profileModelMock as any,
+      messageModelMock as any,
+      {} as any,
+      mockServicesPreferencesModel as any,
+      aPastOptOutEmailSwitchDate
+    );
+
+    const result: StoreMessageContentActivityResult = await storeMessageContentActivityHandler(
+      mockContext,
+      {
+        ...aCreatedMessageEvent,
+        message: {
+          ...aNewMessageWithoutContent,
+          senderServiceId: "01234567890" as ServiceId
+        }
+      }
+    );
+
+    expect(result.kind).toBe("FAILURE");
+    if (result.kind === "FAILURE") {
+      expect(result.reason).toEqual("SENDER_BLOCKED");
+    }
+
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalled();
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalledTimes(1);
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalledWith([
+      aRetrievedProfile.fiscalCode
+    ]);
+
+    const documentId = makeServicesPreferencesDocumentId(
+      aRetrievedProfile.fiscalCode,
+      "01234567890" as NonEmptyString,
+      manualProfileServicePreferencesSettings.version as NonNegativeInteger
+    );
+    expect(findServicePreferenceMock).toHaveBeenCalled();
+    expect(findServicePreferenceMock).toHaveBeenCalledTimes(1);
+    expect(findServicePreferenceMock).toHaveBeenCalledWith([
+      documentId,
+      aRetrievedProfile.fiscalCode
+    ]);
+  });
+
+  it("should return a failure if message sender service exists in profile.blockedInboxOrChannels (LEGACY SETTINGS)", async () => {
```

Comment:
is this the same scenario tested in [here](https://github.com/pagopa/io-functions-services/pull/142/files?diff=split&w=1#diff-3d5477f69edf2d4434f5f726aef090d871dffe3b10040fb4e72a62da72cb2e8fR510)?

Reply:
Clicking the link brings me to the "changed files" section. I cannot see which test you refer to.

Reply:
Tomorrow I will check LEGACY tests. Initially they started different, but it's feasible that some have converged to the same test during development.

Reply:
Yes, no problem ;)
Thanks

Reply:
Removed in https://github.com/pagopa/io-functions-services/pull/142/commits/b9d58101e54264cc46f6fa6221efdc7febeba3e8